src/components/Header/index.tsx

```diff
@@ -391,45 +391,47 @@ export default function Header() {
             </Dropdown>
           </HoverDropdown>
 
-          <HoverDropdown active={pathname.toLowerCase().includes('pools')}>
-            <Flex alignItems="center">
-              <Trans>Earn</Trans>
-              <DropdownIcon />
-            </Flex>
-            <Dropdown>
-              <StyledNavLink
-                id={`pools-nav-link`}
-                to="/pools"
-                isActive={(match, { pathname }) => Boolean(match) || pathname.startsWith('/pools')}
-                style={{ width: '100%' }}
-              >
-                <Trans>Pools</Trans>
-              </StyledNavLink>
+          <Flex id={TutorialIds.EARNING_LINKS} alignItems="center">
+            <HoverDropdown active={pathname.toLowerCase().includes('pools')}>
+              <Flex alignItems="center">
+                <Trans>Earn</Trans>
+                <DropdownIcon />
+              </Flex>
+              <Dropdown>
+                <StyledNavLink
+                  id={`pools-nav-link`}
```

Comment:
so _cồng kềnh_ :joy:

Reply:
Just want to swap 2 links, so I can highlight all of them. any good solution here bro?
![image](https://user-images.githubusercontent.com/33005392/185071207-7fa824c1-0ced-43a1-bdf1-aa2bfdcabac9.png)

Reply:
no, I meant ``id={`pools-nav-link`}`` part. It should be `id='pools-nav-link'`

Reply:
I don't update that line. anyway, I think this is `autocompletion` make it. 

same. when I type `pad`. It suggests me `padding`, when we enter => it fills for us `padding={}`. I think it's is ok. No problem.
<img width="166" alt="image" src="https://user-images.githubusercontent.com/33005392/185073320-96dcb797-4ad6-4e9f-9547-c4cac65c4e29.png">

Reply:
because `padding={}` expecting to receive number value here. For string value case, we dont need and shouldnt keep `{}`

Reply:
There's a lint rule for this. 
`"react/jsx-curly-brace-presence": "warn"`

Reply:
updated.